README.md

```diff
@@ -124,6 +124,44 @@ export default {
   for mapping test type, experience and tool to test code. Not required.
   Defaults to `./d2l-test-reporting.config.json`.
 
+### D2L test reporting configuration file
```

Comment:
You can probably just call this `###Configuration`

Reply:
That's much better. ty